### DISCUSSION TITLE
Slim pane borders how to?

### DISCUSSION THREAD

**Discussion by Quotique** [zellij-org/zellij#763] [Q&A] [answered]
Hi!

I'm looking for setting to enable slim pane borders. Like in this [screenshot](https://zellij.dev/documentation/img/layout-template-example-1-tab-3.png). Please help me

**Comment by imsnif** [comment, marked as answer]
Do you mean to remove the frames? `ctrl+p` + `z` - or add `no_pane_frames: true` to the config

**Comment by Quotique** [comment]
Thank you!

Now I see it in the documentation too.
